toc.go

```diff
@@ -39,7 +39,8 @@ const IndexFormatVersion = 16
 // 6: Include '#' into the LineFragment template
 // 7: Record skip reasons in the index.
 // 8: Record source path in the index.
-const FeatureVersion = 8
+// 9: Store ctags metadata
+const FeatureVersion = 9
```

Comment:
aah good catch. Last minute thought, but any chance we can store the ctags data after everything else is written such that reading the data in older zoekt just works? Then we won't require a reindex.

Reply:
This would be a tool of its own for migrating between Zoekt index versions.

Reply:
aah ok, nevermind then :)